src/EnergyPlus/OutputReportPredefined.cc

```diff
@@ -257,8 +257,10 @@ namespace OutputReportPredefined {
         s->pdstMech = newPreDefSubTable(state, s->pdrEquip, "Central Plant");
 
         s->pdchMechType = newPreDefColumn(state, s->pdstMech, "Type");
-        s->pdchMechNomCap = newPreDefColumn(state, s->pdstMech, "Nominal Capacity [W]");
-        s->pdchMechNomEff = newPreDefColumn(state, s->pdstMech, "Nominal Efficiency [W/W]");
+        s->pdchMechNomCap = newPreDefColumn(state, s->pdstMech, "Reference Capacity [W]");
+        s->pdchMechNomEff = newPreDefColumn(state, s->pdstMech, "Reference Efficiency [W/W]");
+        s->pdchMechRatCap = newPreDefColumn(state, s->pdstMech, "Rated Capacity [W]");
+        s->pdchMechRatEff = newPreDefColumn(state, s->pdstMech, "Rated Efficiency [W/W]");
```

Comment:
These changes add new output columns in tabular output for Standard Rated conditions *Capacity* and *COP*.

Reply:
This results in tabular output file change.

Reply:
New columns in existing table output should be described in "\src\Transition\OutputRulesFiles\OutputChanges9-5-0-to-9-6-0.md". I've also added the OutputChange label to this PR and moved this to the 9.6 I/O Freeze milestone.

Reply:
OK. I am going to update the *OutputChanges9-5-0-to-9-6-0.md* file.